src/azure-cli/azure/cli/command_modules/resource/_params.py

```diff
@@ -166,13 +226,16 @@ def load_arguments(self, _):
 
     with self.argument_context('provider') as c:
         c.ignore('top')
-        c.argument('resource_provider_namespace', options_list=['--namespace', '-n'], completer=get_providers_completion_list, help=_PROVIDER_HELP_TEXT)
+        c.argument('resource_provider_namespace', options_list=[
+                   '--namespace', '-n'], completer=get_providers_completion_list, help=_PROVIDER_HELP_TEXT)
 
     with self.argument_context('provider register') as c:
         c.argument('mg', help="The management group id to register.", options_list=['--management-group-id', '-m'])
-        c.argument('accept_terms', action='store_true', is_preview=True, help="Accept market place terms and RP terms for RPaaS. Required when registering RPs from RPaaS, such as 'Microsoft.Confluent' and 'Microsoft.Datadog'.", deprecate_info=c.deprecate(hide=True))
+        c.argument('accept_terms', action='store_true', is_preview=True,
+                   help="Accept market place terms and RP terms for RPaaS. Required when registering RPs from RPaaS, such as 'Microsoft.Confluent' and 'Microsoft.Datadog'.", deprecate_info=c.deprecate(hide=True))
         c.argument('wait', action='store_true', help='wait for the registration to finish')
-        c.argument('consent_to_permissions', options_list=['--consent-to-permissions', '-c'], action='store_true', help='A value indicating whether authorization is consented or not.')
+        c.argument('consent_to_permissions', options_list=[
+                   '--consent-to-permissions', '-c'], action='store_true', help='A value indicating whether authorization is consented or not.')
```

Comment:
Why do you format so much irrelevant code? Please note that this will increase the cost of our reviewing. Please revert these unnecessary changes first

Reply:
I missed a commit which removed the unnecessary formatting. I pushed the commit and the formatting should be removed. Thanks for being patient.